### PR TITLE
Adding BackupManagerBundle

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| License       | MIT

Adding a little recipe for [backup manager](https://github.com/backup-manager/symfony).

### CODE REVIEW

**Comment by Nyholm** [comment]
Ping @xabbuh, could you give this a review?

**Comment by Nyholm** [comment]
Ping @stof, could you review this?

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
@symfony-flex-server review please

File: backup-manager/symfony/2.1/config/packages/backup_manager.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# See full config at https://github.com/backup-manager/symfony
+bm_backup_manager:
```
Comment (xabbuh) [inline backup-manager/symfony/2.1/config/packages/backup_manager.yaml]: Just wondering whether the file must not (should not) be named `bm_backup_manager.yaml` then.

File: backup-manager/symfony/2.1/config/packages/backup_manager.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    database:
+        development:
+            dsn: '%env(DATABASE_URL)%'
+    storage:
+        local:
+            type: Local
+            root: "%kernel.cache_dir%/db"
```
Comment (xabbuh) [inline backup-manager/symfony/2.1/config/packages/backup_manager.yaml]: Use single quotes here too?

Comment (xabbuh) [reply backup-manager/symfony/2.1/config/packages/backup_manager.yaml]: By the way, using the cache directory (which may be cleared frequently) is probably not a good idea, is it?

File: backup-manager/symfony/2.1/config/packages/backup_manager.yaml (hunk 1 above)
Comment (Nyholm) [reply backup-manager/symfony/2.1/config/packages/backup_manager.yaml]: Thank you

File: backup-manager/symfony/2.1/config/packages/backup_manager.yaml (hunk 2 above)
Comment (Nyholm) [reply backup-manager/symfony/2.1/config/packages/backup_manager.yaml]: You would probably configure a "off site location" for your backup. This is a configuration that will always work and that the user would probably use this config when testing the bundle's functinality.

**Comment by Nyholm** [comment]
Could I get another review on this?

**Comment by Nyholm** [comment]
You are correct. Thanks. 

https://github.com/backup-manager/symfony/issues/48

**Pull request merged**